src/main/java/com/cloudbees/jenkins/plugins/sshagent/exec/ExecRemoteAgent.java

```diff
@@ -135,7 +132,7 @@ public void stop() throws IOException, InterruptedException {
     /**
      * Parses ssh-agent output.
      */
-    private Map<String,String> parseAgentEnv(String agentOutput) throws Exception{
+    private Map<String,String> parseAgentEnv(String agentOutput, TaskListener listener) throws Exception{
```

Comment:
Why? Seems unused.

Reply:
It's used on line 140 and 144, or am I missing something?

Reply:
Sorry, saw a new method parameter without a corresponding body change and forgot that this was switching from a field reference to a parameter reference.